lib/ronin/exploits/exploit.rb

```diff
@@ -556,15 +556,28 @@ def perform_cleanup
       #
       # @since 1.0.0
       #
-      def exploit(dry_run: false)
-        perform_build
+      def exploit(dry_run: false, test: false)
+        if test
+          case result = self.test
+          when TestResult::Vulnerable
+            print_positive "Vulnerable: #{result.to_s}"
+          when TestResult::NotVulnerable
+            print_negative "NotVulnerable: #{result.to_s}"
+          when TestResult::Unknown
+            print_warning "Unknown: #{result.to_s}"
+          else
+            print_error "Unexpected result: #{result}"
```

Comment:
Maybe add an `.inspect` to `result` to properly show what kind of object was returned.